easybuild/easyconfigs/t/Tkinter/Tkinter-3.7.4-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Tkinter'
+version = '3.7.4'
+
+homepage = 'http://python.org/'
+description = """Tkinter module, built with the Python buildsystem"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/python/%(version)s/']
+sources = ['Python-%(version)s.tgz']
+checksums = ['d63e63e14e6d29e17490abbe6f7d17afb3db182dbd801229f14e55f4157c4ba3']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('libffi', '3.2.1'),
+]
+
+dependencies = [
+    ('Python', version),
+    ('Tk', '8.6.9'),
```

Comment:
This is probably going to be the package, but, we still need Tk (and X11, Perl, etc. )